e2e/playwright/projects.spec.ts

```diff
@@ -2014,6 +2032,10 @@ test(
       await expect(page.getByTestId('app-theme')).toHaveValue('dark')
 
       await page.getByTestId('app-theme').selectOption('light')
+      await expect(page.getByTestId('app-theme')).toHaveValue('light')
+
+      // Give time to system for writing to a persistent store
+      await page.waitForTimeout(1000)
```

Comment:
Oof we gotta fix the persistence to signal that it's done, but this is a good find in the meantime.